app/MMB/src/scenes/tripServices/hotelMenuItem/HotelMenuItem.js

```diff
@@ -0,0 +1,150 @@
+// @flow strict
+
+import * as React from 'react';
+import { Translation, DeviceInfo } from '@kiwicom/mobile-localization';
+import {
+  TextIcon,
+  Dimensions,
+  type DimensionType,
+} from '@kiwicom/mobile-shared';
+import {
+  MenuItem,
+  withNavigation,
+  type NavigationType,
+} from '@kiwicom/mobile-navigation';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import idx from 'idx';
+
+import LocationItem from './LocationItem';
+import LocationsPopup from '../LocationsPopup';
+import type { HotelMenuItem as HotelMenuItemType } from './__generated__/HotelMenuItem.graphql';
+import BookingDetailContext from '../../../context/BookingDetailContext';
+
+type State = {|
+  isPopupVisible: boolean,
+|};
+
+type PropsWithContext = {|
+  ...Props,
+  +currency: string,
+  +dimensions: DimensionType,
+|};
+
+class HotelMenuItem extends React.Component<PropsWithContext, State> {
+  state = {
+    isPopupVisible: false,
+  };
+
+  navigateToHotels = (cityId: string, cityName: string) => {
+    this.props.navigation.navigate('MMBHotelsStack', {
+      bookingComAffiliate: '', // TODO
+      language: DeviceInfo.getLanguage(),
+      currency: this.props.currency,
+      dataSaverEnabled: false, // TODO
+      checkin: '2018-09-10', // TODO
+      checkout: '2018-09-27', // TODO
+      onNavigationStateChange: () => {}, // TODO
+      onBackClicked: null, // TODO:
+      dimensions: this.props.dimensions,
+      version: 'rn-development', // TODO
+      cityName,
+      cityId,
+      roomsConfiguration: [{ adultsCount: 1, children: [] }],
```

Comment:
Since you could get this information from querying passengers, this would be TODO as well, right?

Reply:
yes